### ISSUE TITLE
Python support?

### ISSUE THREAD

**Issue by lukoshkin** [antosha417/nvim-lsp-file-operations#38]
Hey!
Does this work in Python?
I tried but didn't manage to make it work

**Comment by antosha417** [comment]
Hey @lukoshkin! 
Which language server have you tried? 
Does it support file operations?

**Comment by lukoshkin** [comment]
I tried pyright and pylsp. But that experiment was quite dirty. Since they interfere with each other. I will try new tests later. Particularly I will try to find lsp that support file operations
@antosha417

**Comment by KevinNitroG** [comment]
@antosha417 I tried with `basedpyright` and it doesn't work either. It shows

```
Could not cut [file] - EPERM: operation not permitted: [source] -> [dest]
```

edit: On my linux it can move, but it doesn't trigger lsp's capability. Other files still maintain their imports

**Comment by JakobGM** [comment]
I managed to get `basedpyright` file renames to work, but not moving the files, with [this config](https://github.com/JakobGM/dotfiles/commit/0fc90405f1e1b077c4fd0de21a8768ef269adc79).

**Comment by sghng** [comment]
Yep, it works with any server that implements workspace.fileOperations, including basedpyright (which is in the README now). Nothing per-language needed on our side. Closing, but if a specific server misbehaves, open an issue with the details.
